Parsers/SysmonParser.txt

```diff
@@ -1,5 +1,5 @@
 // KQL Sysmon Event Parser
-// Last Updated Date: Sept 4, 2019
+// Last Updated Date: Nov 25, 2019
 // Sysmon Version : 10.2.0.0
```

Comment:
Change Sysmon Version to 10.41, also, lets add in the Schema Version of the lastest - schemaversion="4.22"